python/shotgun_model/shotgun_model.py

```diff
@@ -472,6 +489,7 @@ def _load_data(self, entity_type, filters, hierarchy, fields, order=None, seed=N
         params_hash.update(str(self.__fields))
         params_hash.update(str(self.__order))
         params_hash.update(str(self.__hierarchy))
+        params_hash.update(str(self.__column_fields))
```

Comment:
You're right.  This came from when I was storing more in the cache before the easy path presented itself.  Removed.